internal/servers/controller/handlers/targets/target_service.go

```diff
@@ -34,7 +35,18 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
-var maskManager handlers.MaskManager
+var (
+	maskManager     handlers.MaskManager
+	TargetIdActions = action.Actions{
```

Comment:
mind adding godoc comment for new exported var?

Reply:
It's only exported because the test is in a `_test` package so I don't intend for others to use it, but I can.